frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.unit.spec.ts

```diff
@@ -309,4 +310,39 @@ describe("dataset transform functions", () => {
       expect(result).toEqual([]);
     });
   });
+
+  describe("getDatasetExtents", () => {
+    const keys = ["series1", "series2"];
+
+    test("should return correct extents for each series", () => {
```

Comment:
```suggestion
    it("should return correct extents for each series", () => {
```

Use `it` instead of `test`